src/adaptations/device-layer/include/Weave/DeviceLayer/EFR32/gatt_db.h

```diff
@@ -0,0 +1,49 @@
+/***************************************************************************/ /**
```

Comment:
This file needs an OpenWeave licence header.

Reply:
Done